RELEASENOTES.md

```diff
@@ -6,6 +6,96 @@ This document contains the release notes for `rippled`, the reference server imp
 
 Have new ideas? Need help with setting up your node? [Please open an issue here](https://github.com/xrplf/rippled/issues/new/choose).
 
+# Version 2.2.0
+
+Version 2.2.0 of `rippled`, the reference server implementation of the XRP Ledger protocol, is now available. This release adds the price oracle, performance optimizations, several a bug fixes, and introduces the `featurePriceOracle`, `fixEmptyDID`, `fixXChainRewardRounding`, `fixPreviousTxnID`, and `fixAMMv1` amendments.
```

Comment:
nits:
```suggestion
Version 2.2.0 of `rippled`, the reference server implementation of the XRP Ledger protocol, is now available. This release adds the price oracle, performance optimizations, several bug fixes, and introduces the `featurePriceOracle`, `fixEmptyDID`, `fixXChainRewardRounding`, `fixPreviousTxnID`, and `fixAMMv1` amendments.
```

Price Oracle is mentioned twice in this sentence here, might be redundant

Reply:
Yeah, I was listing out all the amendments, but I also wanted to highlight the price oracle. But I agree it doesn't read well. I took out the first reference. Fixed (force pushed).